### PR TITLE
#168389920 Get all mentors

### PR DESCRIPTION
**Pull request by BurnerB**
#### What does this PR do?
Allows auser to view all  mentors
#### Description of Task to be completed?
* Create the get all mentor endpoint
* Ensure tests pass
#### How should this be manually tested?
* Clone this  repository https://github.com/BurnerB/FreeMentors-V2.git and cd into FreeMentors-V2
* checkout to branch ft-DB-get-all-mentors-168389920 
* Run npm --init to install all dependancies.
* Run command `npm test` on your terminal
* Tests should pass
* Run the command `npm start` and access g`/api/v1/mentors` :
* You should recieve a response body if the data  is present:
```
"status": 200,
    "data": [
        {
            "id": 2,
            "firstname": "John",
            "lastname": "Doe",
            "email": "johndoe6@email.com",
            "address": "Nairobi Kenya",
            "bio": "rapper, record producer, and actor who was known as one of the most-controversial and best-selling artists of the early 21st century",
            "occupation": "Musician",
            "expertise": "rapping",
            "ismentor": true,
            "isadmin": false,
            "created_on": "2019-09-10T03:25:20.407Z"
        }
    ]
```
or
```
{
    "status": 404,
    "error": "No mentors"
}
```
if no mentors are present
#### What are the relevant pivotal tracker stories?
[#168389920](https://www.pivotaltracker.com/story/show/168389920)
#### Screenshots (if appropriate) 

get all mentors

### CODE REVIEW

File: server/routes/userRoutes.js [hunk 1]
```diff
@@ -11,6 +11,6 @@ const router = express.Router();
 router.post('/auth/signup', [alreadyRegistered], Validations.validateSignup, Users.registerUser);
 router.post('/auth/signin', Validations.validateLogin, Users.userLogin);
 router.get('/mentors', Mentors.getAllMentors);
-router.get('/mentors/:mentorId', Mentors.getSpecificMentors);
+// router.get('/mentors/:mentorId', Mentors.getSpecificMentors);
```
Comment (ekumamait) [inline server/routes/userRoutes.js]: commented code should not be pushed to production

File: server/tests/admin.test.js [hunk 2]
```diff
@@ -1,142 +1,133 @@
...
+dotenv.config();
 chai.should();
 chai.use(chaiHttp);
 
 let userToken;
 let mentorToken;
 let adminToken;
 
-describe('ADMIN', () => {
+describe('MENTOR', () => {
```
Comment (ekumamait) [inline server/tests/admin.test.js]: Are we testing the Admin or Mentor?

**Review by ekumamait — COMMENTED** [review]
Take a look at the inline comments

File: server/tests/admin.test.js (hunk 2 above)
Comment (BurnerB) [reply server/tests/admin.test.js]: it should be Admin sorry i'll  fix it

**Comment by ekumamait** [comment]
LGTM

**Pull request merged**